app/Services/ParticipantService.php

```diff
@@ -34,5 +34,14 @@ public function addGoodParticipantsToParticipantsTable($goodParticipants){
 
 
     }
+
+    public function removeUserFromParticipantsTable($user) {
+        $userParticipant = Participant::where('user_id', $user['user_id'])->first();
+
+        if( $userParticipant != null){
```

Comment:
Shouldn't this also delete the user from the `user_groups` and `participants` table?

Reply:
We may want to keep that info later on, once someone no longer has a matching a participant_id, they can no longer log in to the app